package.json

```diff
@@ -8,8 +8,6 @@
     "bootstrap": "^4.0.0",
     "d3-selection": "^1.3.2",
     "d3-zoom": "^1.7.3",
-    "eslint-config-airbnb": "^16.1.0",
-    "eslint-plugin-react": "^7.5.1",
```

Comment:
I agree on removing these since we don't use them directly on this project right now.

But I think it's important to have a linting system available for all our projects. For now, I'll just reinforce the issue that is already open on this matter
- #58

Reply:
react-scripts has eslint as a dependency, so it is already installed, we just need a configuration, which can be done with a package.json key `eslintConfig`.
The best option would be to use the react-scripts default configuration but it may be extended for custom rules.